tensorflow/tools/benchmark/benchmark_model.h

```diff
@@ -38,7 +38,8 @@ Status InitializeSession(int num_threads, const string& graph,
 // Does a single run of the model that's been loaded into the given session.
```

Comment:
maybe add a comment saying when step_stats is requested, the inference_time returned might be pessimistic?